.gitattributes

```diff
@@ -4,21 +4,4 @@ test/sambah/reference_files/all_data.nc4 filter=lfs diff=lfs merge=lfs -text
 test/sambah/reference_files/spatial_only.nc4 filter=lfs diff=lfs merge=lfs -text
 test/sambah/reference_files/temp_var_bbox.nc4 filter=lfs diff=lfs merge=lfs -text
 test/sambah/reference_files/var_only.nc4 filter=lfs diff=lfs merge=lfs -text
-test/opera-rtc-s1-browse/reference_data/*.png filter=lfs diff=lfs merge=lfs -text
-test/hybig/reference_data/**/*.png filter=lfs diff=lfs merge=lfs -text
-test/hybig/reference_data/**/*.jpg filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_fland_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_nobs_40km_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_nobs_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_sss_smap_40km_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_gice_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_gland_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_sss_ref_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_sss_smap_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_sss_smap_uncertainty_reformatted.tif filter=lfs diff=lfs merge=lfs -text
-test/net2cog/reference_data/RSS_smap_SSS_L3_8day_running_2015_136_FNL_v04.0_surtep_reformatted.tif filter=lfs diff=lfs merge=lfs -text
 test/smap-l2-gridder/reference_files/**/*.nc filter=lfs diff=lfs merge=lfs -text
-test/net2cog/** !text !filter !merge !diff
-test/hybig/** !text !filter !merge !diff
-test/opera-rtc-s1-browse/** !text !filter !merge !diff
-test/regridder/** !text !filter !merge !diff
```

Comment:
You should leave these, these are explicit excludes I think.

Reply:
Fair - [414eb6b](https://github.com/nasa/harmony-regression-tests/pull/169/commits/414eb6b91007125d75a43cb3f10df6cdd618d5fc).